discoveryengine/search.js

```diff
@@ -57,10 +57,20 @@ async function main(
       servingConfig: name,
     };
 
+    const IResponseParams = {
+      ISearchResult: 0,
+      ISearchRequest: 1,
+      ISearchResponse: 2,
+    };
```

Comment:
Is there a built in enum in the client library for this?

If so, use it instead.

Reply:
I can't find any. Will have to go with this first and update later if I can find better options.